### PR TITLE
Change directory structure to be a proper role

### PR DESCRIPTION
**Pull request by machacekondra**
This PR is changing the structure of the repo to be a proper role:

  - It moves the helper modules to `module_utils` directory, so user don't need to set those as env `ANSIBLE_MODULE_UTILS` variable.
 - It moves the modules to `library` directory, so user don't need to set those as `ANSIBLE_LIBRARY` variable.

Now it's OK if user just use the role before any other use of the modules, without setting the env vars.

For example:

```yaml
---
- name: Load modules
  hosts: localhost

  roles:
    - role: kubevirt.kubevirt-modules
      install_python_requirements: no

- hosts: localhost

  tasks:
    - name: Fetch VM 'myvm'
      kubevirt_facts:
        name: myvm
        namespace: vms
        kind: VirtualMachine
```

Signed-off-by: Ondra Machacek <omachace@redhat.com>

### CODE REVIEW

**Comment by machacekondra** [comment]
Please review, @pkliczewski  @masayag  @vatsalparekh

**Comment by alexxa** [comment]
@machacekondra @pkliczewski @vatsalparekh Please mind that the repo structure you want to change corresponds to the repo structure of [Ansible Git project](https://github.com/ansible/ansible/tree/devel/lib/ansible/modules) and @tripledes made it that way, so the files will be placed in a PR to Ansible Git repo accordingly.

**Comment by alexxa** [comment]
@machacekondra I'm OK whether you merge it or not =). I have just shared why it was done that way.

File: tests/units/modules/test_kubevirt_vm_status.py [hunk 1]
```diff
@@ -7,7 +7,7 @@
 from ansible.module_utils._text import to_bytes
 
 # FIXME: paths/imports should be fixed before submitting a PR to Ansible
```
Comment (masayag) [inline tests/units/modules/test_kubevirt_vm_status.py]: Does this change solve the FIXME ?

Comment (machacekondra) [reply tests/units/modules/test_kubevirt_vm_status.py]: Not yet, I am working on it in another patch.

**Comment by pkliczewski** [comment]
@machacekondra please rebase and I will merge

**Comment by machacekondra** [comment]
Closing based on discussion here: https://github.com/kubevirt/ansible-kubevirt-modules/pull/105

**Pull request closed without merging**